src/franchise.js

```diff
@@ -58,7 +59,19 @@ class Voter {
    }
 }
 
+function computeVoteValue(voteBuffer) {
+   const voteValueHash = crypto.createHash("sha256")
```

Comment:
No, `keccak256` is not in the spec, it is with `sha256`. The reasoning behind it is in https://github.com/vocdoni/docs/blob/master/src/architecture/protocol/anonymous-voting/zk-census-proof.md#zkinputs-generation:
>  - Since the encoded vote values may not fit into a constant number of circuit inputs, we calculate a summary of the raw user vote using an EVM-friendly hash function: `sha256(vote_bytes)`.  The output of the sha256 hash is slightly larger than the field used in SNARKS, so we split the hash output (32-bytes) into 2 16-byte arrays, take them as integers (in little-endian), and use them as circuit inputs.
>    - `sha256` hash is used, as if necessary in the future it can be [verified inside](https://github.com/iden3/circomlib/blob/master/circuits/sha256/sha256.circom) the circuit. This usage has two characteristics to keep in mind:
>      - `sha256` is twice as expensive as `keccak256` in terms of gas in EVM, but it is implemented in `circom`, so it can be checked inside a circuit
>      - checking the `sha256` inside a circom circuit is expensive in terms of number of constraints (in the current version of this spec, this is not checked inside the circuit)
>  - example:
>    ```go
>    h := sha256.Sum256(voteBytes) // voteBytes can be the votes array converted to bytes, or the encrypted votes
>    b1 := new(big.Int).SetBytes(swapEndianness(h[:16])) // swap endianness, as golang big int package works in big-endian, and we use little-endian
>    b2 := new(big.Int).SetBytes(swapEndianness(h[16:]))
>    ```
>    And the json input of the `voteValue` for the circuit would be: `"voteValue": [b1, b2]`

Anyway, currently we are not chekcing the hash inside the circuit so does not affect at the final number of constraints, but if at some point in the future is needed we can include the check inside the circuit, as `sha256` is implemented in circomlib. And `sha256` is twice expensive than `keccak256` in terms of gas in EVM, but as we are not using EVM in this use case it is not a problem.